components/common_safe_work_procedures.mdx

```diff
@@ -0,0 +1,24 @@
+Clearpath Robotics recommends the following procedures be followed to reduce the likelihood of harm.
+
+- Maintain a safe distance of at least ten meters from the robot whenever possible.
+- Never assume that the robot can be reliably stopped on command.
+- Assert a stop using the remote stop device (if present) before approaching the robot.
+- Never place the remote stop device (if present) on or inside the robot.
+- Always be vigilant around the robot, even when a stop is asserted.
+- Perform no maintenance or troubleshooting on the robot without training and familiarisation.
+- Follow electrical safety best practices when working with electrical systems and components.
+- Disconnect battery using main disconnect of the robot before working inside robot.
+- Wear personal protective equipment including high-visibility clothing, head and eye protection while
+  operating, observing or working on the robot.
+- Never climb on or ride the robot.
+- Never lie on the ground under or near the robot.
+- Avoid operating the robot on slopes and grades.
+- Never stand downhill of the robot.
+- Operate the robot at the lowest speed possible.
+- Never attempt to defeat a safety function for any reason.
+- Chock the robot wheels when the robot will be stationary for more than a few seconds.
+- Never work on the robot on a slope.
+- Replace fuses only with identical product from specified manufacturer.
+- Discontinue use and contact Clearpath Robotics support at the first sign of strange robot behaviour.
+- Keep tools and loose parts away from electrical components and out of the robot's bays.
+- Check the functionality of stop systems on a regular basis and before and after each use.
```

Comment:
Change to:
_Check the functionality of stop systems on a regular basis, including before and after each use of the robot._

Reply:
Done